simulation/navigator_gazebo/models/wamv/wamv.sdf

```diff
@@ -34,71 +34,61 @@
           </mesh>
         </geometry>
       </collision>
```

Comment:
It all looks good to me, just check with @DSsoto to make sure the *clip*, *noise*, and *fov* parameters line up with the actual hardware and that the *pose* of each camera is correct so that tf can work it's magic.